src/e2e/java/teammates/e2e/util/LNPResultsStatistics.java

```diff
@@ -0,0 +1,69 @@
+package teammates.e2e.util;
+
+import org.testng.Reporter;
+
+/**
+ * Represents the L&P test results statistics.
+ */
+public class LNPResultsStatistics {
+
+    private double errorPct;
+    private double meanResTime;
+    private double pct1ResTime; // 90th percentile
+    private double throughput;
+    private int errorCount;
+    private int sampleCount;
+    private String resultsErrorMessage = "";
+
+    /**
+     * Verify the LNP results statistics with the specified threshold.
+     */
+    public void verifyLnpTestSuccess(double errorRateLimit, double meanResTimeLimit) {
```

Comment:
Continue from my point, by SRP principle, `Statistics` should not be responsible to check limit.